src/components/VolunteerSideView.jsx

```diff
@@ -0,0 +1,247 @@
+import { Flex, Button, Image, Text, HStack, Box, VStack, IconButton } from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import { useState, useEffect } from 'react';
+import { Icon } from '@chakra-ui/react';
+import { getEventById } from '../utils/eventsUtils';
+import { EditIcon, CalendarIcon } from '@chakra-ui/icons';
+import logos_google_calendar from '../assets/logos_google-calendar.svg';
+import logos_google_maps from '../assets/logos_google-maps.svg';
+import { IoPeopleSharp } from 'react-icons/io5';
+import { IoMdLink } from 'react-icons/io';
+
+const VolunteerSideView = ({ eventId }) => {
+  const [eventData, setEventData] = useState([]);
+  const [isReadMore, setIsReadMore] = useState(false);
+  const [calendarSelected, setCalendarSelected] = useState(false);
+  const [mapSelected, setMapSelected] = useState(false);
+
+  useEffect(() => {
+    getEventById(eventId).then(data => setEventData(data));
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'eventId'. Either include it or remove the dependency array.